contributing/releasing.md

```diff
@@ -38,32 +38,37 @@ Releasing is important enough that we want to start with a clean slate:
 
 ### Cut a release branch and notify clients
 
-Verify that there are no existing release-candidate branches either locally or on origin:
-
-    git fetch -a
-    git remote prune origin
-    git branch -a | grep release-candidate
```

Comment:
This is caught by the `cut` script, so it's not necessary for the release engineer to run this.

Reply:
I think that was @randallli, I usually skip it because I keep my branches pretty clean.